<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Firma</class>
 <widget class="QMainWindow" name="Firma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firma</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>1101</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>PÅ™ehled</string>
     </attribute>
     <widget class="QListWidget" name="lw_novy">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>251</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="l_novy">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Novi</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_schvalen">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>251</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_prijat">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>251</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="lw_zamitnut">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>60</y>
        <width>251</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="l_schvalen">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Schvaleni</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_prijat">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prijati</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_zamitnut">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamitnuti</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_n_dale">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dale</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_n_zamitnout">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>zamitnout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_s_dale">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dale</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_s_zamitnout">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>zamitnout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Detail</string>
     </attribute>
     <widget class="QLabel" name="l_obrazek">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>411</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="l_jmeno">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Jmeno</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_vek">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vek</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_bydliste">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bydliste</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_schopnosti">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Schopnosti</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_stav">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>30</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>STAV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="l_posledniZmena">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>100</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Posl. zmena</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
